src/System.Management.Automation/engine/Utils.cs

```diff
@@ -762,17 +762,17 @@ private static T GetPolicySettingFromConfigFile<T>(ConfigScope[] preferenceOrder
                     PolicyBase result = null;
                     switch (typeof(T).Name)
                     {
-                        case nameof(ScriptExecution):             result = policies.ScriptExecution; break;
-                        case nameof(ScriptBlockLogging):          result = policies.ScriptBlockLogging; break;
-                        case nameof(ModuleLogging):               result = policies.ModuleLogging; break;
-                        case nameof(ProtectedEventLogging):       result = policies.ProtectedEventLogging; break;
-                        case nameof(Transcription):               result = policies.Transcription; break;
-                        case nameof(UpdatableHelp):               result = policies.UpdatableHelp; break;
+                        case nameof(ScriptExecution): result = policies.ScriptExecution; break;
```

Comment:
would it be better to have the statements on seperate lines?

```c#
case nameof(ScriptExecution):
  result = policies.ScriptExecution;
  break;
```

Reply:
Fixed.